app/reports/subscription_collection_report.rb

```diff
@@ -0,0 +1,45 @@
+class SubscriptionCollectionReport
+  def initialize(subscriptions)
+    @subscriptions = subscriptions.map do |subscription|
+      SubscriptionReport.new(subscription, columns)
+    end
+  end
+
+  def to_csv
```

Comment:
@carloslopes No, it depends of how many subscriptions a project has. It could be few or a huge number of subscriptions.